scenarios/test_library_conf.py

```diff
@@ -0,0 +1,41 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import context, BaseTestCase, interfaces, released, bug, irrelevant, missing_feature
+from utils.tools import logger
+
+# PHP JAVA ok
+
+
+@irrelevant(library="cpp")
+class Test_HeaderTags(BaseTestCase):
```

Comment:
Could you add `coverage` decorator ? Reminder : 

* `@coverage.basic` => most simple test case, it ensures at least that the main use case of the feature is working
* `@coverage.complete` => You are confortable to says that a bug can't happen if the test is sucessful (at least for contexts covered by your weblogs)
* `@coverage.good` => everything else